modules/spring-config/src/test/java/org/mule/runtime/config/internal/factories/FlowRefFactoryBeanTestCase.java

```diff
@@ -262,7 +262,7 @@ public void dynamicFlowRefDoesNotExist() throws Exception {
   private FlowRefFactoryBean createFlowRefFactoryBean(String name) throws Exception {
     FlowRefFactoryBean flowRefFactoryBean = new FlowRefFactoryBean();
     flowRefFactoryBean.setName(name);
-    flowRefFactoryBean.setAnnotations(singletonMap(LOCATION_KEY, fromSingleComponent("flow")));
+    flowRefFactoryBean.setAnnotations(singletonMap(LOCATION_KEY, DefaultComponentLocation.from("flow")));
```

Comment:
static import

Reply:
there's already Flux.from (or Mono.from, done really remember now) statically imported in this class